### ISSUE TITLE
Have you implemented all the functions of superpoweredSDK? What about Android?

### ISSUE THREAD

**Issue by themakerman** [MathieuDebit/react-native-superpowered#1]
Hey mate,

FIrst of all it was really awesome to see someone do this implementation. I was going to take this in my hands but thankfully its not needed now. I was wondering if you have implemented functions like waveform visualization as well?

**Comment by MathieuDebit** [comment]
Hey!

Thank you, actually I didn't implemented everything, but only the DJ App example. It's all I was needed, on iOS, but I think it's enough to understand how to create a React Native module, and more over with Superpowered.

Today I don't need this anymore, but if you do, you could fork this project, or open a PR, I'll be glad to take a look or help you!

**Comment by themakerman** [comment]
But did you tested it with Android?

**Comment by MathieuDebit** [comment]
No I didn't. But the implementation shouldn't be difficult with what I already did, as soon as you know how to run C++ on Android.

**Comment by grahamlyus** [comment]
@themakerman @MathieuDebit Spent the afternoon making it work on Android

https://github.com/MathieuDebit/react-native-superpowered/pull/2

**Comment by themakerman** [comment]
@grahamlyus what did u implemented?
All the native functions as RN modules?

**Comment by grahamlyus** [comment]
@themakerman Yes, I've implemented the same 4 native functions present in the IOS module: `init`, `playPause`, `fxValue` and `crossFader`. These are implemented in `SuperpoweredModule.java`.

**Comment by jamesharvey2** [comment]
@grahamlyus wow, would you please upload your android version of this SDK?

**Comment by grahamlyus** [comment]
@jamesharvey2 It's in the pull request, #2
